cranelift/frontend/src/ssa.rs

```diff
@@ -622,7 +618,7 @@ impl SSABuilder {
                     jt = func.create_jump_table(copied);
                 }
 
-                // Redo the match from `analyze_branch` but this time capture mutable references
+                // Redo the match from, but this time capture mutable references
                 match &mut func.dfg.insts[branch] {
                     InstructionData::BranchTable {
                         destination, table, ..
```

Comment:
It's worth noting that once `br_table` uses `BlockCall`, we can replace the outer match with a mutable match, and remove this inner re-match.